dhis-2/dhis-support/dhis-support-test/src/main/resources/tracker/simple_metadata.json

```diff
@@ -530,7 +530,7 @@
       "lastUpdated": "2015-03-31T11:22:51.642",
       "name": "Height in cm",
       "shortName": "Height in cm",
-      "formName": "Height in cm",
+      "formName": "height in cm",
```

Comment:
Why is that necessary? I would expect real data to read `"Height in cm"` in the UI

Reply:
I'm not testing how it's shown in the UI, I'm just testing that even if the form name starts with a lowercase character, it's sorted correctly when I get the change logs.